va/meson.build

```diff
@@ -268,6 +286,53 @@ if WITH_WAYLAND
     dependencies : deps)
 endif
 
+if WITH_WIN32
+  libva_win32_sources = [
+    'win32/va_win32.c',
+  ]
+
+  libva_win32_headers = [
+    'win32/va_win32.h',
+  ]
+
+  libva_headers_subproject += libva_win32_headers
+
+  deps = [ libwin32_dep, libva_dep]
+
+  libva_win32_args = [
+    '-DLIBVA_MAJOR_VERSION=@0@'.format(libva_major_version)
+  ]
+
+  install_headers(libva_win32_headers, subdir : 'va')
+
+  libva_exports = 'libva.def'
+  libva_exports_path = '@0@/@1@'.format(meson.current_source_dir(), libva_exports)
```

Comment:
This isn't needed I'm pretty sure, you should be able to just pass `'libva.def'` to `vs_module_defs`.